src/misc.jl

```diff
@@ -66,7 +66,7 @@ lstirling_asym(x::BigFloat) = loggamma(x) + x - log(x)*(x - big(0.5)) - log2π/b
 lstirling_asym(x::Integer) = lstirling_asym(float(x))
 
 const lstirlingF64 = Float64[lstirling_asym(k) for k in big(1):big(64)]
-const lstirlingF32 = Float64[lstirling_asym(k) for k in big(1):big(40)]
+const lstirlingF32 = Float32.(view(lstirlingF64, 1:40))
```

Comment:
We could avoid the broadcasting machinery here and use something simpler (for the compiler at least) such as
```suggestion
const lstirlingF32 = [Float32(lstirlingF64[i]) for i in 1:40]
```